keyboards/era/sirind/tomak/tomak.c

```diff
@@ -0,0 +1,182 @@
+            RGB_MATRIX_INDICATOR_SET_COLOR(97, rgb_caps.r, rgb_caps.g, rgb_caps.b);
+            RGB_MATRIX_INDICATOR_SET_COLOR(98, rgb_caps.r, rgb_caps.g, rgb_caps.b);
+        } else if (g_tomak_config.indicator_override) {
+            RGB_MATRIX_INDICATOR_SET_COLOR(96, 0, 0, 0);
+            RGB_MATRIX_INDICATOR_SET_COLOR(97, 0, 0, 0);
+            RGB_MATRIX_INDICATOR_SET_COLOR(98, 0, 0, 0);
+        }
+    }
+
+    return true;
+}
+
+bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
+    switch(keycode) {
+        case IN_TOGG:
+        {
+            if (record->event.pressed) {
+                g_tomak_config.indicator_toggle = g_tomak_config.indicator_toggle ? false : true;
+                write_tomak_config_to_eeprom(&g_tomak_config);
+            }
+            return false;
+        }
+        case IN_OVER:
+        {
+            if (record->event.pressed) {
+                g_tomak_config.indicator_override = g_tomak_config.indicator_override ? false : true;
+                write_tomak_config_to_eeprom(&g_tomak_config);
+            }
+            return false;
+        }
+        case IN_BRI:
+        {
+            if (record->event.pressed) {
+                g_tomak_config.indicator_hsv.v = increment( g_tomak_config.indicator_hsv.v, 8, 0, 255 );
+                write_tomak_config_to_eeprom(&g_tomak_config);
+            }
+            return false;
+        }
+        case IN_BRD:
+        {
+            if (record->event.pressed) {
+                g_tomak_config.indicator_hsv.v = decrement( g_tomak_config.indicator_hsv.v, 8, 0, 255 );
+                write_tomak_config_to_eeprom(&g_tomak_config);
+            }
+            return false;
+        }
+        case IN_HUEI:
+        {
+            if (record->event.pressed) {
+                g_tomak_config.indicator_hsv.h = increment( g_tomak_config.indicator_hsv.h, 8, 0, 255 );
+                write_tomak_config_to_eeprom(&g_tomak_config);
+            }
+            return false;
+        }
+        case IN_HUED:
+        {
+            if (record->event.pressed) {
+                g_tomak_config.indicator_hsv.h = decrement( g_tomak_config.indicator_hsv.h, 8, 0, 255 );
+                write_tomak_config_to_eeprom(&g_tomak_config);
+            }
+            return false;
+        }
+        case IN_SATI:
+        {
+            if (record->event.pressed) {
+                g_tomak_config.indicator_hsv.s = increment( g_tomak_config.indicator_hsv.s, 8, 0, 255 );
+                write_tomak_config_to_eeprom(&g_tomak_config);
+            }
+            return false;
+        }
+        case IN_SATD:
+        {
+            if (record->event.pressed) {
+                g_tomak_config.indicator_hsv.s = decrement( g_tomak_config.indicator_hsv.s, 8, 0, 255 );
+                write_tomak_config_to_eeprom(&g_tomak_config);
+            }
+            return false;
+        }
+    }
+    return process_record_user(keycode, record);
+};
```

Comment:
This custom indicator code should be removed.
```suggestion
```

Reply:
@waffle87 
Is the most recently modified 'tomak.c' acceptable?